client/views/root/canada_post.js

```diff
@@ -0,0 +1,3 @@
+import RootView from '../generic-settings-form';
```

Comment:
Is there a way to do this without having a specific root view for each service? This means if we want to add a new shipping service to WooCommerce Connect we'd have to roll the client.  We want to avoid that.